java-backend/src/main/java/org/kframework/backend/java/symbolic/KILtoSMTLib.java

```diff
@@ -571,10 +587,14 @@ public SMTLibTerm transform(KItem kItem) {
         if (label.startsWith("(")) {
             // smtlib expression instead of operator
             String expression = label;
+            boolean hasBinder = recordBinders(label, kList);
```

Comment:
This whole feature is extraordinary error-prone. I'd say a warning must be added here when it is used.

Reply:
added

Reply:
This is no longer applicable.